src/promptflow-tracing/pyproject.toml

```diff
@@ -36,17 +36,20 @@ packages = [
 
 [tool.poetry.dependencies]
```

Comment:
make promptflow depends on this package? and dependencies from there?

Reply:
will add this once we get b2 released

Reply:
Let's get this PR merged

Reply:
made promptflow depende on this, let's wait for the ci